client/src/components/Contests/Filter.jsx

```
@@ -98,77 +96,84 @@ function Filter() {
         >
           <FormControl
             variant="filled"
             sx={{ m: 1, minWidth: 300 }}
-            className={"filter bg-filter rounded-lg platform-container max-sm:justify-center"} // to make it fixed while scroll add class "fixed" on condition "isFixed"
+            className={"filter bg-filter rounded-lg platform-container max-sm:justify-center"}
           >
             <InputLabel
               variant="filled"
               id="platform-select-label"
-              shrink={selectedPlatforms.length == 0 ? false : true}
+              shrink={selectedPlatforms.length === 0 ? false : true}
```

Biome

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)